source/extensions/filters/http/jwt_authn/token_cache.cc

```diff
@@ -0,0 +1,33 @@
+#include "extensions/filters/http/jwt_authn/token_cache.h"
+
+using std::chrono::system_clock;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+TokenCache::TokenCache(int cache_size)
+    : SimpleLRUCache<std::string, ::google::jwt_verify::Jwt>(cache_size) {}
+
+TokenCache::~TokenCache() { clear(); }
+
+::google::jwt_verify::Jwt* TokenCache::find(const std::string& token, bool& cache_hit) {
```

Comment:
do we need cache_hit argument?  can we just use function return?  if return nullptr, it is cache miss

Reply:
Done.